### PR TITLE
PMM-12293 doc update

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: docs/release-notes/2.38.1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Percona Monitoring and Management 2.38.1
+
+| **Release date:** | July 11, 2023                                                                                    |
```
Comment (rnovikovP) [inline docs/release-notes/2.38.1.md:3]:
```suggestion
| **Release date:** | July 12, 2023                                                                                    |
```
I think tomorrow - is a more realistic date.

File: docs/release-notes/2.38.1.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) With this update, we are fixing an authentication issue that was causing PMM to incorrectly display a "User already exists" error when trying to re-login into PMM using a Percona Account.
```
Comment (rnovikovP) [inline docs/release-notes/2.38.1.md:11]: The error is no a problem. the problem - inability to login with PErcona Account

**Review by rnovikovP — CHANGES_REQUESTED** [review]
The description of the bug sounds like it's a error problem, while the issue is bigger. 

Maybe this is worth a release highlight, explaining the issue  and  in Fixed issues - juts a jira id and title

File: docs/release-notes/2.38.1.md (hunk 2 above)
Comment (BupycHuk) [reply docs/release-notes/2.38.1.md]: @rnovikovP should we also mention how to fix it for instances with other identity providers affected?

Comment (rnovikovP) [reply docs/release-notes/2.38.1.md]: yes, we need to mention this 100%

**Review by Nailya — APPROVED** [review]

File: docs/release-notes/2.38.1.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+| **Release date:** | July 12, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) - With this update, we are fixing an authentication issue that occurred when trying to relog into PMM using a Percona Account. Second and subsequent logins no longer result in a "User already exists" error.
```
Comment (rnovikovP) [inline docs/release-notes/2.38.1.md:11]:
```suggestion
[PMM-12293](https://jira.percona.com/browse/PMM-12293) - With this update, we have fixed the "User already exists" error: an authentication issue that occurred when users tried to log back into PMM through a Percona Account. Second and subsequent logins no longer result in this error. 
```

File: docs/release-notes/2.38.1.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) - The upgrade to Grafana  9.2.20 in PMM 2.38 fixed critical security [CVE-2023-3128](https://grafana.com/blog/2023/06/22/grafana-security-release-for-cve-2023-3128/) by changing the authentication mechanism to rely solely on the user's unique ID provided by the identity provider.
+
+This change caused an authentication issue for second and subsequent connections to PMM using a Percona Account. With this PMM update, we are fixing this authentication issue to ensure that the "User already exists" error is no longer incorrectly displayed when re-logging into PMM.
```
Comment (BupycHuk) [inline docs/release-notes/2.38.1.md]:
```suggestion
This change caused an authentication issue for second and subsequent logins to PMM using a Percona Account. With this PMM update, we are fixing this authentication issue to ensure that the user can re-login and the "User already exists" error is no longer incorrectly displayed when re-logging into PMM.
```

**Review by BupycHuk — APPROVED** [review]

File: docs/release-notes/2.38.1.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) - The upgrade to Grafana  9.2.20 in PMM 2.38 fixed critical security [CVE-2023-3128](https://grafana.com/blog/2023/06/22/grafana-security-release-for-cve-2023-3128/) by changing the authentication mechanism to rely solely on the user's unique ID provided by the identity provider.
+
+This change caused an authentication issue for second and subsequent connections to PMM using a Percona Account. With this PMM update, we are fixing this authentication issue to ensure that the "User already exists" error is no longer incorrectly displayed when users log back into PMM.
+
```
Comment (rnovikovP) [inline docs/release-notes/2.38.1.md:14]: But "User already exists"  was a "proper" error. The Core problem for the user - They can't Log in to the PMM at all. The error message they see is secondary.

**Pull request merged**